backend/FwLite/MiniLcm.Tests/AutoFakerHelpers/EntryFakerHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿using MiniLcm.Models;
+using Soenneker.Utils.AutoBogus;
+
+namespace MiniLcm.Tests.AutoFakerHelpers;
+
+public static class EntryFakerHelper
+{
+    public static async Task<Entry> EntryReadyForCreation(this AutoFaker autoFaker,
+        IMiniLcmApi api,
+        Guid? entryId = null,
+        bool createComplexForms = true,
+        bool createComplexFormTypes = true,
+        bool createComponents = true)
+    {
+        var entry = autoFaker.Generate<Entry>();
+        if (entryId.HasValue) entry.Id = entryId.Value;
+        if (createComponents) await CreateComplexFormComponentEntry(entry.Id, true, entry.Components, api);
+        if (createComplexForms) await CreateComplexFormComponentEntry(entry.Id, false, entry.ComplexForms, api);
+        if (createComplexFormTypes) await CreateComplexFormTypes(entry.ComplexFormTypes, api);
+        foreach (var sense in entry.Senses)
+        {
+            sense.EntryId = entry.Id;
```

Comment:
I don't want to have to think about this when using our faker, but we should write a test and decide what we want to happen if the EntryId does not match on the sense, maybe if it's empty then we set it, but if it's not the same should we throw?

Reply:
I think we discussed this a bit a while back. Senses don't move from entry to entry.
So, I think your suggestions makes a lot of sense.

Reply:
created #1236 to track this and similar validation issues